app/message/write/_components/form.tsx

```diff
@@ -30,21 +30,35 @@ const MessageWriteForm = ({
       {
         id: 'content',
         validate: value => value.length > 0,
-        message: '내용을 입력해주세요',
+        message: ' ',
+      },
+      {
+        id: 'content',
+        validate: value => value.length <= 200,
+        message: '200자 이내로 입력해주세요',
       },
     ],
   });
 
+  const isDisabled =
+    Object.keys(errors).length === 0 ||
+    errors?.content?.length > 0 ||
+    isPending;
```

Comment:
혹시 content의 빈 값을 직접 확인하는 것도 가능할까요?
```suggestion
  const {content} = values
  const isDisabled = content==='' ||
    isPending;
```

Reply:
허 이게 훨씬 직관적이고 간단하네요!! 수정해보겠습니다!

Reply:
수혁님~! 그런데 이렇게 수정하면 더 생각해봐야 할 부분이 있는 것 같아요!

입력 글자 200자 제한시 isDisabled하는 방법인데
- 200자 초과인 경우 input에서 추가로 입력이 안되게 하는 방법 (early return)
- 현재처럼 유효성 검사 메시지로 확인하는 방법

아래에서 제안드린 0/200 이 방법은 첫번째 방법과 더 어울리는 것 같아요.

만약 두번째 방법으로 한다면, 제안드린 코드를 다시 생각해봐야 할 것 같아요

Reply:
첫번째 방법이 괜찮을것 같네요! 한번 적용 해보겠습니다.!🙂